apis/goharbor.io/v1beta1/harbor_types.go

```diff
@@ -509,6 +509,11 @@ type HarborStorageImageChartStorageSpec struct {
 	// An implementation of the storagedriver.StorageDriver interface which uses Google Cloud for object storage.
 	// See https://docs.docker.com/registry/storage-drivers/gcs/
 	Gcs *HarborStorageImageChartStorageGcsSpec `json:"gcs,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// An implementation of the storagedriver.StorageDriver interface which uses Google Cloud for object storage.
+	// See https://docs.docker.com/registry/storage-drivers/oss/
+	AliOSS *HarborStorageImageChartStorageAliOSSSpec `json:"aliOss,omitempty"`
```

Comment:
Naming question: we generally use the same name as docker distribution to name storage drivers. In this case it would be oss.
Why prefixing it by Ali ?

Reply:
well, `oss` seems too general, so I add Ali.
But if this is traditional, I will fix this.